qemu/tests/x86_cpu_model.py

```diff
@@ -19,11 +21,18 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
+    qemu_binary = utils_misc.get_qemu_binary(params)
+    cmd = 'echo -e \'{ "execute": "qmp_capabilities" }\n' \
```

Comment:
```suggestion
    qmp_cmds = ['{"execute": "qmp_capabilities"}',
                '{"execute": "query-cpu-definitions", "id": "RAND91"}',
                '{"execute": "quit"}']
    cmd = "echo -e '{0}' | {1} -qmp stdio -vnc none -M none | grep return | grep RAND91".format(r"\n".join(qmp_cmds), qemu_binary)
```
Just a suggestion to make cmd more readable, and the ID `RAND91` also can let it more meaningful.

Reply:
Updated the cmd but keep the id "RAND91", as I don't want to name it as something related to cpu, which might be confusing.